docs/src/main/sphinx/connector/mariadb.md

```diff
@@ -299,6 +299,12 @@ statements, the connector supports the following features:
 ```{include} sql-delete-limitation.fragment
 ```
 
+(mysql-fte-support)=
+## Fault-tolerant execution support
+
+The connector supports {doc}`/admin/fault-tolerant-execution` of query
+processing. Read and write operations are both supported with any retry policy.
+
```

Comment:
Could you update `fault-tolerant-execution.md`? The page mentions the supported connector names.

Reply:
Done d46219abbe